Matrix.SynapseInterop.Worker.FederationSender/Program.cs

```diff
@@ -32,9 +35,17 @@ private static void Main(string[] args)
             
             ServicePointManager.DefaultConnectionLimit = _config.GetSection("Http").GetValue("connectionLimit", 50);
 
+            Console.CancelKeyPress += new ConsoleCancelEventHandler(OnExit);
+
             new FederationSender(_config).Start().Wait();
 
-            Console.ReadKey(true);
+            _closing.WaitOne();
+        }
+
+        protected static void OnExit(object sender, ConsoleCancelEventArgs args)
+        {
+                Console.WriteLine("Exit");
+                _closing.Set();
```

Comment:
The indentation on these 2 lines is wrong

Reply:
done